### ISSUE TITLE
Drawer and deep linking

### ISSUE THREAD

**Issue by razvanpaul** [FaridSafi/react-native-gifted-chat#350]
I use this amazing library in coalition with [react-native-navigation](https://github.com/wix/react-native-navigation). It seems to work fine in some cases, even if I had to implement some handlers. See #270 and #244.

I have the following: Homepage, Drawer, Match and finally the Chat. At the initial moment of the app, I present the homepage with the drawer in the left side, where I choose the Match page. The Match is instantiated using deep links, `onNavigatorEvent` and `this.props.navigator.resetTo`.

Finally, when an item is tapped in the Match page, I want the app to go to the Chat page, but all I get is this:
![simulator screen shot 27 jan 2017 15 01 27](https://cloud.githubusercontent.com/assets/8782666/22371741/7fac34cc-e4a1-11e6-9c80-003d13a3fde2.png)

I use `this.props.navigator.push` to do it, but I've also tried `this.props.navigator.showModal`, nothing worked.
Observation: Everything is good when I firstly instantiate the Match page, instead of the Homepage. Apparently something is not ok with react-native-navigation and deep linking here.

Can you tell me any solution please?

* React Native version: 0.39
* react-native-gifted-chat version: 0.0.10
* Platform(s) (iOS, Android, or both?): iOS

**Comment by razvanpaul** [comment]
Update

Apparently, this is the difference when the Match page is firstly instantiated and when it is not:

Instantiated:
![simulator screen shot 27 jan 2017 18 58 52](https://cloud.githubusercontent.com/assets/8782666/22379814/9b7644fa-e4c3-11e6-9e58-25e0f66b89b8.png)


Not instantiated:
![simulator screen shot 27 jan 2017 19 02 49](https://cloud.githubusercontent.com/assets/8782666/22379825/a651c82c-e4c3-11e6-8b8a-bd18fac2ee7c.png)

**Comment by kfiroo** [comment]
Hey @razvanpaul 
Can you try to use latest commit from master and see if your issues are solved?

**Comment by razvanpaul** [comment]
I've tried to get 0.0.11, but it says it's unavailable. How can I get it?

**Comment by kfiroo** [comment]
@razvanpaul in your package.json use this:

```
"react-native-gifted-chat": "git+https://github.com/FaridSafi/react-native-gifted-chat#d3387e2f55a6067beec08d21eed4f42402e7779e",
```

**Comment by razvanpaul** [comment]
@kfiroo Thank you!

But, unfortunately, the bug persists. Here is a video:
https://www.dropbox.com/s/nmzzyi6d34dxyn6/recording.mov?dl=0

**Comment by razvanpaul** [comment]
I've also tried #277. It didn't work.

**Comment by razvanpaul** [comment]
Solved thanks to the pull request: #355.
